### DISCUSSION TITLE
7기 프리코스 릴리즈 홍보

### DISCUSSION THREAD

**Discussion by jminkkk** [woowacourse-teams/2024-code-zap#798] [이야기해봐요]
### 회의시간
10월 16일 17시

### 안건

홍보를 어떻게 진행할것인가?
- 정확히 언제 홍보글을 올릴 것인가?
- 홍보글에 어떤 내용을 포함할것인가?
- 이벤트는 어떻게 진행할것인가?

**Comment by Jaymyong66** [comment]
# 6기 + 7기 + 지인 둥둥

-> 홍보글 작성은 누가? 언제까지? 
월하, 다음주 월

# 정확히 언제 홍보글을 올릴 것인가?
> 2024년 10월 23일부터 홍보 가능

### 가이드라인 올라오자마자 바로 시작

# 홍보글에 어떤 내용을 포함할것인가?

이벤트 + 서비스 소개 + 내 템플릿 페이지 or 홈페이지 캡쳐

# 이벤트는 어떻게 진행할것인가?

## 선정기준 (우리에게 이메일은 한번은 보내야함, 이메일 보낼 때 현재 로그인된 memberId를 같이 보낼 수 있음)
1. 작성을 많이 해준 사람
2. 유의미한 피드백을 보내면 랜덤으로 뽑기
3. 작성 하나 이상, 피드백 하나 포함 이메일 보내기

### 열혈유저상
- 작성을 많이 해준 사람
치킨 1명 + 5등까지 커피

### 피드백상
- 랜덤 뽑기
메가커피 5명
